### PR TITLE
Add support for Minecraft server scanning

### PR DESCRIPTION
**Pull request by Mr-Bossman**
Scanning for Minecraft serves is a popular thing to do.
 Add basic support for detecting Minecraft servers.

### CODE REVIEW

**Comment by TudbuT** [comment]
Very cool, would love to see this merged!

File: src/proto-banner1.c [hunk 1]
```diff
@@ -127,10 +129,10 @@ banner1_parse(
                         banner1->smack,
                         &tcb_state->state,
                         px, &offset, (unsigned)length);
-        if (x != SMACK_NOT_FOUND)
-            proto = patterns[x].id;
-        else
+        if (x == SMACK_NOT_FOUND)
             proto = 0xFFFFFFFF;
+        else
+            proto = patterns[x].id;
```
Comment (p-l-) [inline src/proto-banner1.c:135]: What's the point of this change? Is it related to this PR?

Comment (Mr-Bossman) [reply src/proto-banner1.c:135]: Remove a double negative I can drop it.

**Comment by Markusklarcus1** [comment]
How would i use this in my code when using masscan?

**Pull request merged**